tests/test_memory.py

```diff
@@ -17,7 +17,7 @@ def issymbolic(value):
 def isconcrete(value):
     return not issymbolic(value)
 
-
+# python3's unittest does not have this function, so we need to implement it ourselves
 def assertItemsEqual(self, a, b):
```

Comment:
Maybe https://stackoverflow.com/questions/6655724/how-to-write-a-custom-assertfoo-method-in-python